packages/hash/frontend/src/shared/layout/layout-with-header/search-bar.tsx

```diff
@@ -159,19 +154,21 @@ const SearchBarWhenSearchIsEnabled: FunctionComponent = () => {
     }
   }, [displayedQuery, displaySearchInput]);
 
-  const { authenticatedUser } = useAuthenticatedUser();
+  const { authenticatedUser: _ } = useAuthenticatedUser();
 
-  const { data, loading } = useQuery<
-    SearchPagesQuery,
-    SearchPagesQueryVariables
-  >(searchPages, {
-    variables: {
-      accountId: authenticatedUser!.entityId,
-      query: submittedQuery,
-    },
-    skip: !authenticatedUser?.entityId || !submittedQuery,
-    fetchPolicy: "network-only",
-  });
+  const data: any = [];
+  const loading = false;
+  // const { data, loading } = useQuery<
```

Comment:
Is there an asana task we can associate with this? I remember seeing some for search

Reply:
Added reference to the task in a5f80242a582547c9a976c1b885e37721193731c